Find a functionally equivalent code:android.os.Bundle.getFloat:COMMENT
Method Modifier: public      
Comment:/**
 * Returns the value associated with the given key, or 0.0f if
 * no mapping of the desired type exists for the given key.
 *
 * @param key a String
 * @return a float value
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    return getFloat(key, 0.0f);
+    return super.getFloat(key);
 }

